support/dynamics/gp/workflow-2-0-tables-used-with-microsoft-dynamics-gp.md

```diff
@@ -40,7 +40,7 @@ WFI10005 - Workflow Step Instance Users
 
 Below lists each table, what information it holds, and what information is written to the tables along the workflow process.
 
-By default, using Microsoft Dynamics GP 2015 R2 as an example, before Workflow 2.0 is set up at all, only these tables will have any records in them: By default, using Microsoft Dynamics GP 2015 R2 as an example, before Workflow 2.0 is set up at all, only these tables will have any records in them:
+By default, using Microsoft Dynamics GP 2015 R2 as an example, before Workflow is set up at all, only these tables will have any records in them:
```

Comment:
Hi @theley502, can you help double check the changes that I made for this sentence? I also remove "2.0" from lines 77 and 130.

Reply:
you are spot on, we don't need the 2.0 reference anymore, great catch, thanks